test/suite/test_config04.py

```diff
@@ -203,18 +203,48 @@ def test_eviction_dirty_tgt_abs_too_large(self):
                                  'eviction_dirty_target=1G'),
             '/eviction dirty target should not exceed cache size/')
 
-    def test_eviction_dirty_trigggr_abs_too_large(self):
+    def test_eviction_dirty_trigger_abs_too_large(self):
```

Comment:
Good catch. It took me a while to see what the change was here.

Reply:
Code Spell Checker is pretty useful to find this kind of typo!